tests/interactive_test.py

```diff
@@ -0,0 +1,201 @@
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+import json
+from tempfile import gettempdir
+
+from spyql.query import Query
+from spyql import log
+from spyql.utils import join_paths
+
+
+def eq_test_nrows(query, expectation, **kwargs):
+    log.user_debug(f"----")
+    q = Query(query)
+    log.user_debug(f"{q}")
+    res = q(**kwargs)
+    log.user_debug(f"{len(expectation)} vs {len(res)} => {res} ")
+    assert len(res) == len(expectation)
+    # TODO: test actual output against expectation!
```

Comment:
❓ Was this TODO forgotten, or will it be in the scope of a future PR/issue?

Reply:
This was fixed on a following commit: https://github.com/dcmoura/spyql/pull/68/commits/b66e451ab48189263ec85c0477cebe9740c68b33

Ideally this would have be done in the same commit, but there were some cross-dependencies so I opted to leave a refactoring commit. Probably I should have move this file from this commit to the refactoring one and called the later "added lib and file testing". I can still do it but I will have to rewrite the git history.